### ISSUE TITLE
Empty .svg file

### ISSUE THREAD

**Issue by beaker1977** [vmware-clarity/ng-clarity#690]
The svg files for icons script-execute and script-schedule are empty.

**Comment by kevinbuhmann** [comment]
Which icon component?

The `cds-icon` component from `@cds/core`?
I can see both on https://core.clarity.design/foundation/icons/shapes when I search for `script`. Which version of the `@cds/core` package are you using?

The `clr-icon` component from`@clr/icons`?
The `@clr/icons` package is deprecated in favor of `cds-icon` from the `@cds/core` package.

**Comment by beaker1977** [comment]
I agree you can see the icons on that page. The problem becomes when you download the accompanying svg file. That is what is empty (see the files that I've attached). As for the version, I'm using 

    "@cds/core": "^6.4.2",
    "@clr/angular": "^15.3.0",
    "@clr/ui": "^15.3.0",

And yes I am using cds-icon component. 

![script-schedule-line](https://github.com/vmware-clarity/ng-clarity/assets/27329476/f5340fcc-8319-4f05-9c02-53bcb25ae194)
![script-execute-line](https://github.com/vmware-clarity/ng-clarity/assets/27329476/c0c4364b-d122-4a76-9e36-88d9a98cc3a4)

**Comment by kevinbuhmann** [comment]
Ah, I see.
